src/metabase/events/view_log.clj

```diff
@@ -95,9 +96,17 @@
 
 (def ^:private update-dashboard-last-viewed-at-interval-seconds 20)
 
-(defn- update-dashboard-last-viewed-at!* [dashboard-ids]
-  (t2/update! :model/Dashboard :id [:in dashboard-ids]
-              {:last_viewed_at :%now}))
+(defn- update-dashboard-last-viewed-at!* [dashboard-id-timestamps]
```

Comment:
do we have tests for this?

I have written one for update last_used_at here https://github.com/metabase/metabase/blob/24d47437cacec9dece4cd50921d29c7a26183c4f/test/metabase/query_processor/middleware/update_used_cards_test.clj#L73

Reply:
Good callout, just added a test for this.